src/toHex.js

```diff
@@ -20,7 +20,19 @@
  * @return {string}
  */
 function toHex(value) {
-  // write code here
+  const hexadecimalVal = [];
+  const divisor = 16;
+  let newVal = value;
```

Comment:
некорректное название + не сокращай слова
```suggestion
  let currentValue = value;
```